client/components/Item/RelatedItems.vue

```diff
@@ -89,39 +89,54 @@ export default Vue.extend({
       loading: true
     };
   },
-  watch: {
-    item(): void {
-      this.refreshItems();
-    }
+  computed: {
+    ...mapGetters('items', ['getItems', 'getMissingIds'])
   },
-  beforeMount() {
-    try {
-      this.refreshItems();
-    } catch (error) {
-      this.pushSnackbarMessage({
-        message: this.$t('unableGetRelated'),
-        color: 'error'
-      });
+  watch: {
+    'item.Id': {
```

Comment:
Is this really necessary? Afaik, this component isn't used anywhere where the item prop would change (When going between pages, a new one is created).